Gemfile

```diff
@@ -67,15 +66,8 @@ explicitly_require_windows_gems = false
 puppet_gem_location = gem_type(puppetversion)
 # This is not a perfect answer to the version check
 if puppet_gem_location != :gem || puppetversion < '3.5.0'
-  is_x64 = Gem::Platform.local.cpu == 'x64'
-  if is_x64
-    platform(:x64_mingw) do
-      explicitly_require_windows_gems = true
-    end
-  else
-    platform (:mingw) do
-      explicitly_require_windows_gems = true
-    end
+  if Gem::Platform.local.os == 'mingw32'
```

Comment:
So previously we intended to explicitly required gems on Windows if:

* Running a 64-bit Ruby on a 64-bit Windows OS
* Running a 32-bit Ruby on a 32-bit Windows OS
* But not if we're running a 32-bit Ruby on a 64-bit OS

With this change, we will explicitly require gems on Windows all the time.  

It seems that this actually fixes a problem in our prior logic - am I understanding that correctly? @ferventcoder I believe this was the issue you saw in TravisCI --  Puppet < 3.5 + a 32-bit Ruby 1.9.3 installed to a 64-bit OS?

:+1: